test/additionalTests/testVisualizeEpistasis/testVisualizationOfEpistasis.m

```diff
@@ -0,0 +1,7 @@
+% % test file for visualization of epistatic interactions
+load('testInputEpistasisVisualization.mat', 'Coli')
```

Comment:
please, in tests always use readCbModel if you are loading COBRA models, as otherwise functionality might not work on current models since the tested models do not adhere to the model field definitions.